cli/context.go

```diff
@@ -253,6 +253,37 @@ func (c FFSContext) truncTableData(str string, collen int) string {
 	}
 }
 
+func (c FFSContext) PrintPrimaryOutputTSV(data [][]string, columnFilter []int) {
+        if c.Opt.Quiet {
+                return
+        }
+
+        if len(data) == 0 {
+                return
+        }
+
+        lens := make([]int, len(data[0]))
```

Comment:
I think the `lens` calculation here can be removed, in the tabular output it was used for visual alignment, but that doesn't really exist in tsv